app/models/point.rb

```diff
@@ -5,8 +5,6 @@ class Point < ApplicationRecord
  belongs_to :topic
  belongs_to :case
 
- belongs_to :case, optional: true
```

Comment:
Is this not still relevant?

Reply:
No, because we require case on point. Optional means case_id isn't required for point.